src/Components/WebAssembly/Authentication.Msal/src/Models/MsalCacheOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Authentication.WebAssembly.Msal.Models
+{
+    /// <summary>
+    /// Cache options for the msal.js cache.
+    /// </summary>
+    public class MsalCacheOptions
+    {
+        /// <summary>
+        /// Gets or sets the cache location.
+        /// </summary>
+        /// <remarks>
+        /// Valid values are <c>sessionStorage</c> and <c>localStorage</c>.
```

Comment:
Does MSAL itself define this feature of choosing between these locations? I'm guessing so since I don't see the value being used anywhere else in our code.

Reply:
Yes, these just expose their underlying config options.